RoutesApp_ios/repository/network/managers/LineRouteFirebaseManager.swift

```diff
@@ -27,6 +28,17 @@ class LineRouteFirebaseManager: LineRouteManagerProtocol {
             }
         }
     }
+    func getLineRouteByDate(idLine: String, date: Date, completion: @escaping(Result<[LineRouteInfo], Error>) -> Void) {
+        firebaseManager.getDocumentsByParameterContainsDate(type: LineRouteInfo.self, forCollection: .LineRoute, field: "idLine",
```

Comment:
doesn't this should be >= date to get the latest changes?

Reply:
totally rigth i have added the changes